docker/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM debian:stable
+
+RUN apt-get -y update
+RUN apt-get -y install sudo
+
+SHELL ["/bin/bash", "-c"]
+
+RUN sudo -n apt-get -y install gcc \
+                           python3 \
+                           python3-pip
+
+
+RUN sudo pip3 install dask distributed
```

Comment:
It is good practise to clean the cache in the same `RUN` instance as installing packages:
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/

apt has clean cache (or `rm -fr the cache path`), `pip` has a `no cache` parameter